src/renderer/views/Post.vue

```diff
@@ -1,9 +1,6 @@
 <template>
   <div>
-    <div v-if="!isInvidiousAllowed">
-      {{ $t('Channel.Community.Viewing Posts Only Supported By Invidious') }}
```

Comment:
Please remove this unused string from at least the en-US file (as I understand it the decision on other PS has essentially boiled down to, removing it from all files causes too many conflicts m, so instead I have to do a follow up every now and then to get rid of them all in one go), that way people won't have to translate a known unused string anymore.

Reply:
Sorry, I missed your first comment after the new version of youtube.js was released.
In this case, removing this string shouldn't cause many/any conflicts so it should be fine :smile: